plugins/catalog-backend-module-github-org/src/module.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 The Backstage Authors
+ * Copyright 2023 The Backstage Authors
```

Comment:
Leave this as is, it's still based on code initially written in 2022

Reply:
Done in https://github.com/backstage/backstage/pull/20361/commits/cdc61c2ac2580ef3617a577bce983a8388b8818d